app/README.md

```diff
@@ -48,46 +53,53 @@ The Databricks Add-on for Splunk is used to query Databricks data and execute Da
-| Client Id \*| [Auth: Azure Active Directory] Azure Active Directory Client Id from your Azure portal.|
-| Tenant Id \*| [Auth: Azure Active Directory] Databricks Application(Tenant) Id from your Azure portal.|
-| Client Secret \*| [Auth: Azure Active Directory] Azure Active Directory Client Secret from your Azure portal.|
+| Field Name                | Field Description                                                                                                                                                                                                                                  |      Required        |
+| ------------------------  | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------- |
+| Account Name              | Unique name for account.                                                                                                                                                                                                                       |      Yes
+| Databricks Instance       | Databricks Instance URL.                                                                                                                                                                                                                           |      Yes
+| Databricks Cluster Name   | Name of the Databricks cluster to use for query and notebook execution. A user can override this value while executing the custom command.                                                                                                         |      No
+| Authentication Method     | SingleSelect: Authentication via Azure Active Directory or using a Personal Access Token |      Yes
+| Databricks Access Token   | [Auth: Personal Access Token] Databricks personal access token to use for authentication. Refer [Generate Databricks Access Token](https://docs.databricks.com/dev-tools/api/latest/authentication.html#generate-a-personal-access-token) document to generate the access token. |      Yes                                                                                              |
+| Client Id   | [Auth: Azure Active Directory] Azure Active Directory Client Id from your Azure portal.|      Yes
+| Tenant Id   | [Auth: Azure Active Directory] Databricks Application(Tenant) Id from your Azure portal.|      Yes
+| Client Secret  | [Auth: Azure Active Directory] Azure Active Directory Client Secret from your Azure portal.|      Yes
```

Comment:
We may need to rename them to be explicit for Azure as we're getting OAuth support for AWS as well.